src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -845,11 +867,20 @@ Object AsyncRPCOperation_sendmany::perform_joinsplit(
             );
 
     // Generate the proof, this can take over a minute.
-    JSDescription jsdesc(*pzcashParams,
+    boost::array<libzcash::JSInput, ZC_NUM_JS_INPUTS> inputs
```

Comment:
Not with `boost::array` [AFAICT](http://www.boost.org/doc/libs/1_62_0/doc/html/boost/array.html).